### DISCUSSION TITLE
TUI improvement

### DISCUSSION THREAD

**Discussion by MTTMTTMTT** [oh-f1sh/TexasPoorGuy#5] [Ideas]
Write down your ideas about TUI improvement.

If you find difficulty using client or just feel ugly about the UI.
Any ideas is welcomed!

**Comment by MTTMTTMTT** [comment]
Add visual effect to SB & BB player

Emoji or Text Art may be a good choice
<img width="272" alt="image" src="https://github.com/oh-f1sh/TexasPoorGuy/assets/45574959/65ae9935-64d6-47b0-abab-794494fe36ad">

**Comment by MTTMTTMTT** [comment]
房间接受服务端推送 玩家进入房间和退出房间的通知

@Golemstorm

**Reply by MTTMTTMTT** [reply]
1. Refresh chat list. List all chat messages sent.
2. Game message window show a new message that someone just joined.

**Comment by MTTMTTMTT** [comment]
房间增加房间用户列表，区分对局用户和房间用户

@Golemstorm

**Reply by MTTMTTMTT** [reply]
你想看到什么样子的？多一个窗口？还是用文字效果区分？

**Reply by Golemstorm** [reply]
another block to show how many players are there in the room

**Reply by MTTMTTMTT** [reply]
Since no need to care about who is watching the game.
There is no need to see who else is in the room.
You can just identify everyone if there is a clear notice in the message window and player list.

I think this can be solved in:
https://github.com/oh-f1sh/TexasPoorGuy/discussions/5#discussioncomment-9505560

**Reply by Golemstorm** [reply]
i dont think so,
the thing is if the game is not start,that wont be shown the player list,so i dont know how many players joined the room,i need to know the room player list
